acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuDepositSender.java

```diff
@@ -90,8 +90,14 @@ public ValidatorKeystores generateValidatorKeys(
   }
 
   public ValidatorKeystores generateValidatorKeys(int numberOfValidators) {
+    return generateValidatorKeys(numberOfValidators, false);
+  }
+
+  public ValidatorKeystores generateValidatorKeys(int numberOfValidators, boolean isLocked) {
```

Comment:
Let's make them final

Reply:
added a bunch, the class is pretty messy